galsim/inclinedexponential.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+"""@file inclinedexponential.py
+
+InclinedExponential is a class representing an exponential profile inclined to the LOS.
+"""
+
+import galsim
+from galsim import GSObject
+from . import _galsim
+
+
+class InclinedExponential(GSObject):
+    """A class describing an inclined exponential profile.
+
+    The Inclined Exponential surface brightness profile is characterized by three properties: its
+    inclination angle 'i' (where 0 = face-on and pi/2 = edge-on), its scale radius, and its scale
+    height. The 3D light distribution function is:
+
+    I(R,z) = I_0 / (2h_s) * sech^2 (z/h_s) * exp(-R/R_s)
+
+    where z is the distance along the minor axis, R is the radial distance from the minor axis,
+    R_s is the scale radius of the disk, h_s is the scale height of the disk, and I_0 is the central
+    surface brightness of the face-on disk.
+
+    In this implementation, the profile is inclined along the y-axis. This means that it will likely
+    need to be rotated in most circumstances.
+
+    At present, this profile is not enabled for photon-shooting.
+
+    Initialization
+    --------------
+
+    @param inclination      The inclination angle, i
```

Comment:
@rmandelb Okay, I'm getting that error as well now, with larger scale heights. Realistic scale heights are of order ~10% the scale radius, and I didn't start getting the error until it was over ~60% of the scale radius. I'll see if there's an easy fix for this, but this is at the point where it's hardly disk-like, and so perhaps we should just put in a note about limits on the scale_height.

It might also be worth letting scale_height default to 10% of the scale_radius, so users won't enter it themselves unless they're certain they want to (in which case it's less likely they'll choose too-large values).